404.html

```diff
@@ -1,25 +1,14 @@
----
-permalink: /404.html
-layout: default
----
 
-<style type="text/css" media="screen">
-  .container {
-    margin: 10px auto;
-    max-width: 600px;
-    text-align: center;
-  }
-  h1 {
-    margin: 30px 0;
-    font-size: 4em;
-    line-height: 1;
-    letter-spacing: -1px;
-  }
-</style>
+<!--permalink: /404.html
+layout: default
```

Comment:
I think we're at a crossroads with this one. If we remove the `default` layout, then this page will no longer show the header, footer, links, etc. So I think we can either 

1. Keep this page with no layout and keep bootstrap (like you have), but add a link to go back to the home page
2. Find a Bootstrap (or similar) styled theme, use that, then re-style the 404 page.

I'm ok with option 1 since it would be less work and we can get the PR merged for you 🙂